src/spatialref.jl

```diff
@@ -3,10 +3,16 @@
 
 Import a coordinate reference system from a `GeoFormat` into GDAL,
 returning an [`AbstractSpatialRef`](@ref).
+
+The `order` keyword argument can be passed in. `:trad` will use traditional
```

Comment:
Should we add this to the docstrings for `fromWKB`, `fromWKT`, `toWKT`, `toISOWKT`, ... as well?

Reply:
If you mean `importESRI`, `importXML` etc then yes. But I haven't implemented it for geometries due to your earlier concerns.